cortex-m/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+authors = [
+    "The Cortex-M Team <cortex-m@teams.rust-embedded.org>",
+    "Jorge Aparicio <jorge@japaric.io>",
+]
+categories = ["embedded", "hardware-support", "no-std"]
+description = "Low level access to Cortex-M processors"
+documentation = "https://docs.rs/cortex-m"
+keywords = ["arm", "cortex-m", "register", "peripheral"]
+license = "MIT OR Apache-2.0"
+name = "cortex-m"
+readme = "README.md"
+repository = "https://github.com/rust-embedded/cortex-m"
+version = "0.7.4"
+edition = "2021"
+rust-version = "1.59"
+links = "cortex-m"  # prevent multiple versions of this crate to be linked together
+
+[dependencies]
+cortex-m-interrupt-number = "1.0.0"
```

Comment:
```suggestion
cortex-m-interrupt-number = { version = "1.0.0", path = "../cortex-m-interrupt-number" }
```

Reply:
Thanks, done!

Reply:
Ah, of course this lets CI run properly, thanks.